docs/content/v0.6/2.configuration/2.nuxt-config.md

```diff
@@ -215,6 +215,18 @@ type ProviderLocal = {
    * @advanced_array_example { id: 'string', email: 'string', name: 'string', role: 'admin | guest | account', subscriptions: "{ id: number, status: 'ACTIVE' | 'INACTIVE' }[]" }
    */
   sessionDataType?: SessionDataObject,
+    /**
+   * How to extract the authentication-token from the sign-in response.
+   *
+   * E.g., setting this to `/data/user` and returning an object like `{ data: { user: { id:number, name: string } }, status: 'ok' }` from the `getSession` endpoint will
+   * storing the 'User' object typed as the type created via the 'sessionDataType' prop.
+   *
+   * This follows the JSON Pointer standard, see it's RFC6901 here: https://www.rfc-editor.org/rfc/rfc6901
+   *
+   * @default / Access the root of the session response object
+   * @example /data/user  Access the `data/user` property of the session response object
+   */
+  sessionDataResponseTokenPointer?: string
```

Comment:
I think we can move this and the `sessionDataType` into the `SessionConfig` and then use it for all providers, right? 

Defaults would just need to be slightly different per provider:
- pointer defaults: `/`
- session data defaults: https://sidebase.io/nuxt-auth/v0.6/application-side/session-access-and-management#sessiondata

Tghis would be nice as it would unify behavior of all providers + make it way easier to type the return of the `authjs` provider (which was a big pain before). What do you think?